src/cloudservice/azext_cloudservice/action.py

```diff
@@ -0,0 +1,17 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=wildcard-import
+# pylint: disable=unused-wildcard-import
+
+from .generated.action import *  # noqa: F403
```

Comment:
why those files not under azext_cloudservice/generated folder?

Reply:
We have three `action.py`. It is magic from autorest. This one is existing one. Others are "generated" one and "manual" one.